client/components/form.js

```diff
@@ -14,17 +14,19 @@ import { Button } from "@mui/material";
 import { requestAppt } from "@/actions/form";
 
 // define schema
-
 const schema = Joi.object({
-  //! add regex to restrict to alpha only
-  name: Joi.string().alphanum().min(2).max(30).required(),
-  //! update tlds config (throws error if not present)
+  name: Joi.string()
+    .pattern(new RegExp(/^[A-Za-z]+$/))
```

Comment:
FYI this wouldn't be a great regex for names in the real world. Silly example: Björk tries to sign up for our app

Reply:
Ahh that is a good point. I could look into how to deal with that if it's feasible haha

Reply:
Might be a good idea to update this at some point to include common symbols used in names like hyphens, commas, apostrophes, etc.